mmcv/fileio/file_client.py

```diff
@@ -20,18 +21,64 @@ def get_text(self, filepath):
 
 
 class CephBackend(BaseStorageBackend):
-    """Ceph storage backend."""
+    """Ceph storage backend.
 
-    def __init__(self):
+    Args:
+        path_maps (dict|None): path mapping dict from local path to Petrel
+            path. When `path_maps={'src': 'dst'}`, `src` in `filepath` will be
+            replaced by `dst`. Default: None.
+    """
+
+    def __init__(self, path_maps=None):
         try:
             import ceph
+            warnings.warn('Ceph is deprecate in favor of Petrel.',
+                          DeprecationWarning)
```

Comment:
DeprecationWarning seems to be ignored by default. You may just use the default warning type.